src/modules/register/services/reset_index.js

```diff
@@ -0,0 +1,19 @@
+import apiUrl from '@/constants/routes/resetPassword'
+
+import API from "@/services/API"
+
+
+ const resetPassword = async (resetPassword)=>{
+    const config = {
+        headers:{
+            'Content-Type': 'application/json',
+            Authorization: 'Bearer ' + "1bxXVTgMmdgiClqXZ8Rdmg"    
```

Comment:
Why the token is static?

Reply:
it will be made dynamic once it is merged